CHANGELOG.rst

```diff
@@ -19,6 +19,10 @@ Fixed
 
   Contributed by @S-T-A-R-L-O-R-D
 
+* Fixed a bug where calling 'get_by_name' on client for getting key details was not returning any results despite key being stored
```

Comment:
Minor comment, it is normal for changelog entries to include the PR number, can you add a #5677 to the changelog comment. This allows people to look up the PR later to see the changes made for a fix.